package.json

```diff
@@ -17,24 +24,19 @@
     "protractor": "^4.0.9"
   },
   "scripts": {
-    "postinstall": "bower install",
-
+    "postinstall": "npm run copy-libs",
     "update-deps": "npm update",
-    "postupdate-deps": "bower update",
-
+    "postupdate-deps": "npm run copy-libs",
+    "copy-libs": "cpx \"node_modules/{angular,angular-*,html5-boilerplate}/**/*\"  app/lib -C",
```

Comment:
This seems less than ideal to copy dependencies manually, having to update and maintain this inline script whenever a dependency is added, like bootstrap for instance. Perhaps it would be simpler to just go the route of using a build tool like webpack or brunch?

Reply:
I think that in any significant project you would end up with some tool like that, but for angular-seed we are trying to keep the complexity and number of tools to the minimum.